crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -121,15 +123,47 @@ impl Plugin for VisibilityPlugin {
 pub fn calculate_bounds(
     mut commands: Commands,
     meshes: Res<Assets<Mesh>>,
-    without_aabb: Query<(Entity, &Handle<Mesh>), (Without<Aabb>, Without<NoFrustumCulling>)>,
+    without_aabb_or_with_changed_mesh: Query<
+        (Entity, &Handle<Mesh>),
+        (
+            Or<(Without<Aabb>, Changed<Handle<Mesh>>)>,
+            Without<NoFrustumCulling>,
+        ),
+    >,
```

Comment:
This `Query` type is too complex for `clippy`'s likes. It doesn't seem trivial to extract but we could try something like:
```rust
type NoAaabOrMeshChanged = Or<(Without<Aabb>, Changed<Handle<Mesh>>)>;
type NeedsAabbUpdate = (NoAaabOrMeshChanged, Without<NoFrustumCulling>);
```
and then do:
```suggestion
    without_aabb_or_with_changed_mesh: Query<
        (Entity, &Handle<Mesh>),
        NeedsAabbUpdate,
    >,
```
but it didn't seem _obviously_ better :shrug:

Reply:
Agreed, just leave it. We have the type_complexity lint turned off at a project level already.

You can run (most of) CI locally using `cargo run -p ci`.